### PR TITLE
fix ReturnnRasrDumpHDFJob

### PR DESCRIPTION
**Pull request by JackTemaki**
At least for me the old version did not work at all because of some missing parameters and incorrect dataset.

### CODE REVIEW

File: returnn/hdf.py [hunk 1]
```diff
@@ -125,11 +130,32 @@ def __init__(
         :param Path alignment:
         :param int num_classes:
         :param int buffer_size:
-        :param kwargs: parameters for ReturnnDumpHDFJob
+        :param int cpu: number of CPU cores required
+        :param int mem: RAM required
+        :param int file_size: request file space on compute node
+        :param int time: compute time
```
Comment (Atticus1806) [inline returnn/hdf.py:136]: Maybe it would make sense to add units here for these parameters in the docs.

**Review by Atticus1806 — COMMENTED** [review]
Cant really comment on the RASR part looks fine in general, but maybe you could add documentation for `feature_flow` etc. aswell if youre already working on the job.

**Review by michelwi — APPROVED** [review]

**Review by christophmluscher — COMMENTED** [review]
code looks good. I have one point where I am concerned and that is that the created hdf file is directly dumped to a file server...

**Comment by michelwi** [comment]
> code looks good. I have one point where I am concerned and that is that the created hdf file is directly dumped to a file server...

true, but this should be part of a different PR for the underlying DumpHDFJob.

**Comment by christophmluscher** [comment]
OK. that is fine for me :)

**Review by christophmluscher — APPROVED** [review]

**Pull request merged**